pkg/apis/app/v1beta1/probe.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1beta1
+
+// KogitoProbe describes a health check to be performed against a container to determine whether it is
+// alive or ready to receive traffic.
+// +k8s:openapi-gen=true
+type KogitoProbe struct {
```

Comment:
Please use the Kubernetes probe object instead.

Reply:
done